RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusteringSequence_cff.py

```diff
@@ -11,18 +11,32 @@
 particleFlowSuperClusteringSequence = cms.Sequence(particleFlowSuperClusterECAL)
 
 particleFlowSuperClusterHGCal = particleFlowSuperClusterECAL.clone()
+particleFlowSuperClusterHGCalFromMultiCl = particleFlowSuperClusterECAL.clone()
 _phase2_hgcal_particleFlowSuperClusteringSequence = particleFlowSuperClusteringSequence.copy()
 _phase2_hgcal_particleFlowSuperClusteringSequence += particleFlowSuperClusterHGCal
+_phase2_hgcal_particleFlowSuperClusteringSequence += particleFlowSuperClusterHGCalFromMultiCl
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
     particleFlowSuperClusterHGCal,
     PFClusters = cms.InputTag('particleFlowClusterHGCal'),
     useRegression = cms.bool(False), #no HGCal regression yet
     use_preshower = cms.bool(False),
-    PFBasicClusterCollectionEndcap = cms.string(""),   
+    PFBasicClusterCollectionEndcap = cms.string(""),
     PFSuperClusterCollectionEndcap = cms.string(""),
     PFSuperClusterCollectionEndcapWithPreshower = cms.string(""),
     thresh_PFClusterEndcap = cms.double(1.5e-1) # 150 MeV threshold
 )
+
```

Comment:
it should be less verbose and easier to maintain if particleFlowSuperClusterHGCalFromMultiCl is removed from L14 and moved here
with the following
```python
particleFlowSuperClusterHGCalFromMultiCl = particleFlowSuperClusterHGCal.clone()
phase2_hgcal.toModify(
 particleFlowSuperClusterHGCalFromMultiCl,
 PFClusters = 'particleFlowClusterHGCalFromMultiCl'
)
```